### PR TITLE
Fix for exceptions thrown for new miners

### PR DESCRIPTION
**Pull request by locohamster**
Fix for #3.
Logs warnings and exposes default values.

### CODE REVIEW

File: .github/workflows/builds.yml [hunk 1]
```diff
@@ -51,7 +51,6 @@ jobs:
         platforms: linux/amd64,linux/arm64,linux/arm/v7
         push: ${{ github.event_name != 'pull_request' }}
         tags: |
-          tedder42/helium_hotspot_exporter:latest
-          ghcr.io/${{ github.repository_owner }}/helium_hotspot_exporter:latest
+          cloud.canister.io:5000/locohamster/loco_exporter:latest
```
Comment (tedder) [inline .github/workflows/builds.yml:54]: you'll need to remove these changes so I can merge.

File: .github/workflows/docker-image2.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        name: Build and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: |
+            cloud.canister.io:5000/locohamster/loco_exporter
+      - name: Print image url
+        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
```
Comment (tedder) [inline .github/workflows/docker-image2.yml]: this should stay in your fork, not in a PR.

File: helium_hotspot_exporter.py [hunk 3]
```diff
@@ -121,6 +123,14 @@ def get_hotspot(hotspot_address):
   ret = req_get_json(mkurl('hotspots/', hotspot_address))
   if not ret: return
   return ret['data']
+def get_hotspot_rewards(hotspot_address, lastxdays):
```
Comment (tedder) [inline helium_hotspot_exporter.py:126]: add a newline here.

File: helium_hotspot_exporter.py [hunk 4]
```diff
@@ -196,8 +214,10 @@ def stats_for_hotspot(addr, hname):
   HOTSPOT_ONLINE.labels(addr,hname).set(isup)
 
   haz_addr = 0
-  if len(d['status']['listen_addrs']):
+  if d['status']['listen_addrs'] and len(d['status']['listen_addrs']):
```
Comment (tedder) [inline helium_hotspot_exporter.py:217]: what error does this fix? I mean, what value is in there? None?

**Comment by tedder** [comment]
Closing, I think you need to do local development before pushing dev work on an open PR.

File: helium_hotspot_exporter.py (hunk 4 above)
Comment (locohamster) [reply helium_hotspot_exporter.py:217]: yes

File: .github/workflows/docker-image2.yml (hunk 2 above)
Comment (locohamster) [reply .github/workflows/docker-image2.yml]: oh sorry, I was currently working on this. this should have been branch used for PR...

File: .github/workflows/builds.yml (hunk 1 above)
Comment (locohamster) [reply .github/workflows/builds.yml:54]: will do.

**Pull request closed without merging**